<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogEditRunway</class>
 <widget class="QDialog" name="DialogEditRunway">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>957</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Runway Edit</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_14">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>580</width>
        <height>955</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_9">
       <item row="12" column="0">
        <widget class="QLabel" name="label_29">
         <property name="text">
          <string>High End VGSI Offset [m]:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>spinBoxVGSI_X_Hi</cstring>
         </property>
        </widget>
       </item>
       <item row="17" column="0" colspan="3">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>538</width>
           <height>3</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Low End VGSI Offset [m]:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>spinBoxVGSI_X_Lo</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QCheckBox" name="checkBoxDepthSortedBin">
         <property name="text">
          <string>Depth Sorted Bin</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Depth Sorted Bin:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>spinBoxDepthSortedBin</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QRadioButton" name="radioButtonZYX">
         <property name="text">
          <string>ZYX</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Width [m]:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>spinBoxWidth</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Convention:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QComboBox" name="comboBoxRALS_Lo">
         <item>
          <property name="text">
           <string>None</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ALSF-1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ALSF-2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>MALS</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>MALSF</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>MALSR</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QComboBox" name="comboBoxVGSI_Hi">
         <item>
          <property name="text">
           <string>None</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PAPI - P4L</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PAPI - P4R</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>VASI - V4L</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>VASI - V4R</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>VASI - V6L</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>VASI - V6R</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QDoubleSpinBox" name="spinBoxLength">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="maximum">
          <double>9999.989999999999782</double>
         </property>
         <property name="value">
          <double>1000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Low End RALS:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>comboBoxRALS_Lo</cstring>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>High End VGSI:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>comboBoxVGSI_Hi</cstring>
         </property>
        </widget>
       </item>
       <item row="14" column="1">
        <widget class="QCheckBox" name="checkBoxTDZL">
         <property name="text">
          <string>TDZL</string>
         </property>
        </widget>
       </item>
       <item row="13" column="1">
        <widget class="QCheckBox" name="checkBoxRCLS">
         <property name="text">
          <string>RCLS</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QDoubleSpinBox" name="spinBoxVGSI_X_Lo">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="maximum">
          <double>2000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QDoubleSpinBox" name="spinBoxWidth">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="maximum">
          <double>99.989999999999995</double>
         </property>
         <property name="value">
          <double>45.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QComboBox" name="comboBoxRALS_Hi">
         <item>
          <property name="text">
           <string>None</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ALSF-1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ALSF-2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>MALS</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>MALSF</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>MALSR</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="spinBoxDepthSortedBin">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>9</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QRadioButton" name="radioButtonXYZ">
         <property name="text">
          <string>XYZ</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Name:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>lineEditName</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLineEdit" name="lineEditName"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Length [m]:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>spinBoxLength</cstring>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Low End VGSI:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>comboBoxVGSI_Lo</cstring>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="QDoubleSpinBox" name="spinBoxVGSI_X_Hi">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="maximum">
          <double>2000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="3">
        <layout class="QGridLayout" name="gridLayout_10">
         <item row="0" column="0">
          <widget class="QGroupBox" name="groupBox">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="title">
            <string>Position</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_11">
            <item row="0" column="0">
             <widget class="QLabel" name="label_23">
              <property name="text">
               <string>X [m]:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="buddy">
               <cstring>spinBoxPositionX</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QDoubleSpinBox" name="spinBoxPositionX">
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="decimals">
               <number>4</number>
              </property>
              <property name="minimum">
               <double>-9999999.998999999836087</double>
              </property>
              <property name="maximum">
               <double>9999999.998999999836087</double>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_24">
              <property name="text">
               <string>Y [m]:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="buddy">
               <cstring>spinBoxPositionY</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="spinBoxPositionY">
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="decimals">
               <number>4</number>
              </property>
              <property name="minimum">
               <double>-9999999.998999999836087</double>
              </property>
              <property name="maximum">
               <double>9999999.998999999836087</double>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_25">
              <property name="text">
               <string>Z [m]:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="buddy">
               <cstring>spinBoxPositionZ</cstring>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QDoubleSpinBox" name="spinBoxPositionZ">
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="decimals">
               <number>4</number>
              </property>
              <property name="minimum">
               <double>-9999999.998999999836087</double>
              </property>
              <property name="maximum">
               <double>9999999.998999999836087</double>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QGroupBox" name="groupBox_8">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="title">
            <string>Attitude</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_12">
            <item row="0" column="0">
             <widget class="QLabel" name="label_26">
              <property name="text">
               <string>X [deg]:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="buddy">
               <cstring>spinBoxAttitudeX</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QDoubleSpinBox" name="spinBoxAttitudeX">
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="decimals">
               <number>4</number>
              </property>
              <property name="minimum">
               <double>-180.000000000000000</double>
              </property>
              <property name="maximum">
               <double>180.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_27">
              <property name="text">
               <string>Y [deg]:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="buddy">
               <cstring>spinBoxAttitudeY</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="spinBoxAttitudeY">
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="decimals">
               <number>4</number>
              </property>
              <property name="minimum">
               <double>-90.000000000000000</double>
              </property>
              <property name="maximum">
               <double>90.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_28">
              <property name="text">
               <string>Z [deg]:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="buddy">
               <cstring>spinBoxAttitudeZ</cstring>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QDoubleSpinBox" name="spinBoxAttitudeZ">
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="decimals">
               <number>4</number>
              </property>
              <property name="minimum">
               <double>-360.000000000000000</double>
              </property>
              <property name="maximum">
               <double>360.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>High End RALS:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>comboBoxRALS_Hi</cstring>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QComboBox" name="comboBoxVGSI_Lo">
         <item>
          <property name="text">
           <string>None</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PAPI - P4L</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PAPI - P4R</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>VASI - V4L</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>VASI - V4R</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>VASI - V6L</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>VASI - V6R</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="15" column="0" colspan="3">
        <widget class="QGroupBox" name="groupBoxGaps">
         <property name="title">
          <string>Gaps</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_13">
          <item row="0" column="0">
           <widget class="QPushButton" name="pushButtonGapAdd">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="3">
           <widget class="QListWidget" name="listWidgetGaps"/>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="pushButtonGapEdit">
            <property name="text">
             <string>Edit</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="pushButtonGapRemove">
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="16" column="0" colspan="3">
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
      </layout>
      <widget class="QDialogButtonBox" name="buttonBox_2">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>1017</y>
         <width>177</width>
         <height>32</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>scrollArea</tabstop>
  <tabstop>lineEditName</tabstop>
  <tabstop>checkBoxDepthSortedBin</tabstop>
  <tabstop>spinBoxDepthSortedBin</tabstop>
  <tabstop>radioButtonXYZ</tabstop>
  <tabstop>radioButtonZYX</tabstop>
  <tabstop>spinBoxPositionX</tabstop>
  <tabstop>spinBoxPositionY</tabstop>
  <tabstop>spinBoxPositionZ</tabstop>
  <tabstop>spinBoxAttitudeX</tabstop>
  <tabstop>spinBoxAttitudeY</tabstop>
  <tabstop>spinBoxAttitudeZ</tabstop>
  <tabstop>spinBoxLength</tabstop>
  <tabstop>spinBoxWidth</tabstop>
  <tabstop>comboBoxRALS_Lo</tabstop>
  <tabstop>comboBoxRALS_Hi</tabstop>
  <tabstop>comboBoxVGSI_Lo</tabstop>
  <tabstop>comboBoxVGSI_Hi</tabstop>
  <tabstop>spinBoxVGSI_X_Lo</tabstop>
  <tabstop>spinBoxVGSI_X_Hi</tabstop>
  <tabstop>checkBoxRCLS</tabstop>
  <tabstop>checkBoxTDZL</tabstop>
  <tabstop>pushButtonGapAdd</tabstop>
  <tabstop>listWidgetGaps</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogEditRunway</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogEditRunway</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
